launchable/commands/subset.py

```diff
@@ -170,6 +175,25 @@ def default_path_builder(file_name):
                     if path:
                         self.test_paths.append(self.to_test_path(path))
 
+        def get_payload(self, session_id, target, duration):
+            payload = {
+                "testPaths": self.test_paths,
+                "session": {
+                    # expecting just the last component, not the whole path
+                    "id": os.path.basename(session_id)
+                }
+            }
+
+            if target is not None:
+                payload["target"] = target
+            elif duration is not None:
+                payload["goal"] = {
+                    "type": "subset-by-absolute-time",
+                    "duration": duration,
+                }
```

Comment:
The new `goal` property is capable of representing every kind of goal, including the percentage based target, so for consistency don't we want to just get rid of `payload["target"]` altogether and always stick to `payload["goal"]` ?

Reply:
I thought that, and there might be no problem, but changing request field is a little big change.
so I thought that we should release gradually.

Next, I'll change this field and release.